packages/block-library/src/rss/index.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Server-side rendering of the `core/rss` block.
+ *
+ * @package gutenberg
```

Comment:
This needs to be WordPress the same as here:
https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/latest-posts/index.php#L5

It's not clear but this is because it gets copied to core, it's similar to translations. I'm wondering where we should put such guidelines. I wouldn't know about it if I didn't see changes in the core after Grunt run a few weeks back :)

Reply:
Done